python_modules/airline-demo/airline_demo_tests/test_airflow.py

```diff
@@ -10,36 +12,29 @@
 from .marks import airflow, slow
 
 
-####################################################################################################
-# These tests are "in-memory" because although they use the Airflow APIs to execute dockerized
-# DAG nodes, they don't run through the full Airflow machinery (a separate scheduler and executor
-# process, or even the single-node out-of-process invocation of `airflow test`)
 @slow
 @airflow
-class TestInMemoryAirflow_0IngestExecution:
+class TestAirflowPython_0IngestExecution:
     pipeline = define_airline_demo_ingest_pipeline()
     config = [
         script_relative_path(os.path.join('..', 'environments', 'local_base.yml')),
         script_relative_path(os.path.join('..', 'environments', 'local_airflow.yml')),
         script_relative_path(os.path.join('..', 'environments', 'local_fast_ingest.yml')),
     ]
 
-    def test_airflow_run_ingest_pipeline(self, in_memory_airflow_run, clean_results_dir):
+    def test_airflow_run_ingest_pipeline(self, dagster_airflow_python_operator_pipeline):
         pass
 
 
 @slow
 @airflow
-class TestInMemoryAirflow_1WarehouseExecution:
+class TestAirflowPython_1WarehouseExecution:
     pipeline = define_airline_demo_warehouse_pipeline()
     config = [
         script_relative_path(os.path.join('..', 'environments', 'local_base.yml')),
         script_relative_path(os.path.join('..', 'environments', 'local_airflow.yml')),
         script_relative_path(os.path.join('..', 'environments', 'local_warehouse.yml')),
     ]
 
-    def test_airflow_run_warehouse_pipeline(self, in_memory_airflow_run, clean_results_dir):
+    def test_airflow_run_warehouse_pipeline(self, dagster_airflow_python_operator_pipeline):
```

Comment:
does it need to be this name (redefining outer scope thing?)

Reply:
pylint isn't smart enough to know this is a pytest fixture. we can maybe get it in scope by importing it to conftest instead, but that leaves the test file itself confusing